packages/amplify_flutter/test/amplify_test.dart

```diff
@@ -25,14 +25,11 @@ void main() {
   AmplifyClass amplify;
   String dummyConfiguration = "dummy";
   String amplifyAlreadyConfiguredError =
-      "Amplify has already been configured and re-configuration is not supported. " +
-          "Please use Amplify.isConfigured to check before calling configure again.";
+      "Amplify has already been configured and re-configuration is not supported.";
```

Comment:
Would it make sense to use these constants in the source code where we throw errors as well? Could be in a future PR.

Reply:
Good point, I'll look to add it in the error handling refactor PR.